charts/gardener/controlplane/charts/application/templates/rbac-user.yaml

```diff
@@ -392,6 +392,56 @@ rules:
   verbs:
   - create
 
+# Cluster role setting the permissions for a project serviceaccount admin. It gets bound by a RoleBinding
+# in a respective project namespace.
+# It aggregates all ClusterRoles labeled with rbac.gardener.cloud/aggregate-to-project-saAdmin: "true"
+---
+apiVersion: {{ include "rbacversion" . }}
+kind: ClusterRole
+metadata:
+  name: gardener.cloud:system:project-saAdmin
```

Comment:
Regarding naming, see below.
Can you otherwise add a TODO that we remove the `serviceaccounts` privilege for "normal" admins in a future release?

Reply:
I added the TODOs that will remove privileges over serviceaccounts for project `members` and `viewers`. Not sure if this is the desired state or do we just leave the view permission for both roles